assets/l10n/app_en.arb

```diff
@@ -1,6 +1,6 @@
 {
     "helloWorld": "Hello World!",
-    "@helloWorld": {
-      "description": "The conventional newborn programmer greeting"
-    }
```

Comment:
I think this could be a good part to have, but tbh in android we normally don't add this. How do you think about having a convention for this later for the flutter-chapter? 🤔

Reply:
Sure la, but do you know that `description` use for? 😁